zhaquirks/tuya/ts011f_plug.py

```diff
@@ -1141,3 +1142,71 @@ class Plug_2AC_var03(CustomDevice):
             },
         },
     }
+
+
+class Plug_CB_Metering(EnchantedDevice):
```

Comment:
just a note: Class names are normally `CamelCase` in Python and don't include underscores, but I see that underscores are used everywhere here, so let's keep it like this (until changed completely?)